cheatsheets/Ruby_on_Rails_Cheatsheet.md

```diff
@@ -18,9 +18,15 @@ exec("os command here")
 open("\| os command here")
 ```
 
-While the power of these commands is quite useful, extreme care should be taken when using them in a Rails based application. Usually, its just a bad idea. If need be, a whitelist of possible values should be used and any input should be validated as thoroughly as possible.
+While the power of these commands is quite useful, extreme care should be taken when using them in a Rails based application. Usually, its just a bad idea. If need be, a whitelist of possible values should be used and any input should be validated as thoroughly as possible. When needed, you can escape a string to use it safely in a Bourne shell.
 
-The Ruby Security Reviewer's Guide has a [section on injection](http://code.google.com/p/ruby-security/wiki/Guide#Good_ol%27_shell_injection) and there are a number of OWASP references for it, starting at the top: [Command Injection](https://www.owasp.org/index.php/Command_Injection).
+```ruby
+require "shellwords"
+
+Shellwords.escape(str)
+```
+
+Check out the OWASP guide for more on [Command Injection](https://www.owasp.org/index.php/Command_Injection).
```

Comment:
Should we link the ruby on rail's security guidelines too? https://guides.rubyonrails.org/security.html#command-line-injection

Reply:
Nice find, that seems like a better approach than manually escaping